lib/resources/r4/encounter_patch.yaml

```diff
@@ -111,3 +111,36 @@ operations:
         <li> <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/payment-collection-status?_format=json">Payment Collection Status</a></li>
         <li> <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/estimated-financial-responsibility-amount?_format=json">Estimated Financial Responsibility Amount</a></li>
       </ul>
+
+  - name: add-extension
+    path: /extension/-
+    operation: add
+    type: List of Extension
+    url: http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.extension
+    description: The array of extensions for an encounter.
+    example: |
+      [
+        {
+          "op": "add",
+          "path": "/extension/-",
+          "value":  {
+            "extension": [
+              {
+                "id": "ENCNTR:2822522",
+                "valueString": "ICD-9 Code",
+                "url": "custom-attribute-name"
+              },
+              {
+                "valueString": "test string",
+                "url": "custom-attribute-value"
+              }
+            ],
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+          }
+        }
+      ]
+    note: >
+      <ul>
+        <li> The custom-attribute id field cannot be set.</li>
+        <li> Add operation is currently supported for <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute?_format=json">Custom Attribute Extension</a></li>
```

Comment:
nit: let's put 'currently **only** supported for...'

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/849/commits/82c12687b37d80a0d2e24607c7931ee43adddcbb#diff-a0c53684e260623feb3826256c549a1eeb448b54febcc11c63fa0c15d4e86b20R145